config/default/layout_builder_styles.settings.yml

```diff
@@ -1,5 +1,5 @@
 multiselect: multiple
 form_type: multiple-select
-groups: "padding|Padding\r\nmargin|Margin\r\nsection_type|Section\r\nheading|Heading\r\nbackground|Background\r\nalignment|Alignment"
+groups: "padding|Padding\r\nmargin|Margin\r\nsection_type|Section\r\nheading|Heading\r\nbackground|Background\r\nalignment|Alignment\r\nbanner_type|Banner type\r\nbanner_height|Banner height\r\nheadline_type|Headline type"
```

Comment:
I am thinking about the following changes:
* Banner type => Content orientation (or something like that). My concern is that the options in question aren't going to be clear to the user.
* Banner height => Height
* Headline type => Headline options

Reply:
Those descriptions sound better!